samples/msgext-northwind-inventory-csharp/.gitignore

```diff
@@ -0,0 +1,25 @@
+# TeamsFx files
+build
+appPackage/build
+env/.env.*.user
+env/.env.local
+appsettings.Development.json
+.deployment
+
+# User-specific files
+*.user
+
+# Build results
+[Dd]ebug/
+[Dd]ebugPublic/
+[Rr]elease/
+[Rr]eleases/
+x64/
+x86/
+bld/
+[Bb]in/
+[Oo]bj/
+[Ll]og/
+
+# Notification local store
+.notification.localstore.json
```

Comment:
We should include the following paths:

- `appsettings.json`
- `.vs`
- `__azurite*`
- `__blobstorage__`
- `__queuestorage__`

Reply:
fixed